src/main/java/com/example/demo/ShortenUrlController.java

```diff
@@ -11,25 +13,27 @@
 public class ShortenUrlController {
 
     @Autowired
-    private UrlRepository repository;
+    private ShortenUrlService shortenUrlService;
 
     @GetMapping
-    public String helloWorld(){
-        return "Hello World from Spring Boot";
+    public ResponseEntity<String> helloWorld(){
+        String msg = "Hello World from Spring Boot";
+        HttpHeaders headers = new HttpHeaders();
+        return new ResponseEntity<String>(msg,headers, HttpStatus.OK);
```

Comment:
Instead of this, let's have some kind of heartbeat URL, when you call this method it will return you "UP" or something like that. It should be open URL and move it to separate HeartBeatController clas please